src/commands/install.rs

```diff
@@ -44,6 +44,7 @@ pub enum FrumError {
 
 pub struct Install {
     pub version: Option<InputVersion>,
+    pub configure_opts: Vec<String>,
```

Comment:
I deliberated for a bit on what type to use here (I had started with `Option<Vec<String>>` but it didn't seem the option provided any real benefit. Having a plan vector allowed me to use `args(configure_opts)` without any need for pattern matching which seemed simpler overall.

Reply:
I think you're right.